text/3857-cfg-version.md

```diff
@@ -548,6 +549,17 @@ If the operator is inside the string literal
 - users may stumble and be frustrated with missing features from cargo (do we include all unary and binary operators?)
 - behavior differences with Cargo may be needed due to different use cases but could lead to user bugs and frustration as it might not match what users are familiar with
 
+If we nest `since` inside `version`,
+- If there is a concern with boundary with `since` conditions that aren't alleviated by the discussion else where,
```

Comment:
"elsewhere" is a compound word
```suggestion
- If there is a concern with boundary with `since` conditions that aren't alleviated by the discussion elsewhere,
```